tools/cli/migrate/scripts/04-Add-leaveAt.js

```diff
@@ -0,0 +1,66 @@
+const dotenv = require('dotenv');
+const { Barman } = require('../../../../server/app/models');
+const sequelize = require('../../../../server/bootstrap/sequelize');
+const config = require('../../../../server/bootstrap/config');
+
+async function init() {
+  dotenv.config();
+  config.start();
+  await sequelize.start();
+}
+
+async function setLeaveAtForOldBarmen() {
+  const oldBarmen = await Barman.findAll({
+    where: {
+      active: false,
+    },
+  });
+  const promises = [];
+  oldBarmen.forEach((oldBarman) => {
+    const newOldBarman = oldBarman;
+    newOldBarman.leaveAt = new Date('2018-07-31');
+    promises.push(newOldBarman.save());
+  });
+  await Promise.all(promises);
+}
+
+async function setActiveForBarmen() {
+  const barmen = await Barman.findAll();
+  const promises = [];
+  barmen.forEach((barman) => {
+    const newBarman = barman;
+    newBarman.active = !barman.leaveAt;
+    promises.push(newBarman.save());
+  });
+  await Promise.all(promises);
+}
+
+/* eslint-disable require-jsdoc,no-unused-vars */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    const barmans = await queryInterface.describeTable('barmans');
+    await init();
```

Comment:
Actually, migration should just do sql requests. So you shouldn't use models. Instead, do a SQL request to update the barman table